.mergify.yml

```diff
@@ -26,6 +26,7 @@ queue_rules:
               - "status-success=golangci-lint"
               - "status-success=mod-check"
               - "status-success=lint-extras"
+              - "status-success=commitlint"
```

Comment:
The commit message does not change when rebasing the PR. Once Mergify decides to put it in this queue, the commitlint check has been validated already (with dependabot commits skipped). There is no need to include commitlint here again (like that, DCO could be removed here too).

Reply:
yeah , true.. let me just remove DCO for now and the rule conditions can be validated later..